build.py

```
@@ -39,7 +40,10 @@ def get_deb_extra_depends() -> str:
     return ""
 
 def system2(cmd):
-    exit_code = os.system(cmd)
+    if isinstance(cmd, list):
+        exit_code = subprocess.call(cmd)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)